cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -568,22 +601,37 @@ def test_embed_tower_exec_play_against_amazon(
         initialEstimate: 1/4h
         tags: ansible_embed
     """
+    if provider_credentials.credential_type == "Amazon":
```

Comment:
**Optional** Perhaps a dictionary would clean up this `if`, `elif` structure.

Reply:
Removed it by adding one entity in above `CREDENTIALS` list.